mmv1/products/netapp/volume.yaml

```diff
@@ -0,0 +1,416 @@
+        - :OPLOCKS
+        - :SHOW_SNAPSHOT
+        - :SHOW_PREVIOUS_VERSIONS
+        - :ACCESS_BASED_ENUMERATION
+        - :CONTINUOUSLY_AVAILABLE
+  - !ruby/object:Api::Type::String
+    name: 'unixPermissions'
+    description: |
+      Unix permission the mount point will be created with. Default is 0770. Applicable for UNIX security style volumes only.
+    default_from_api: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      Labels as key value pairs. Example: `{ "owner": "Bob", "department": "finance", "purpose": "testing" }`.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      An optional description of this resource.
+  # We want to get rid of this. Please don't expose.
+  # - !ruby/object:Api::Type::Integer
+  #   name: 'snapReserve'
+  #   description: |
+  #     Snap_reserve specifies percentage of volume storage reserved for snapshot storage. Default is 0 percent. Use is deprecated.
+  - !ruby/object:Api::Type::Boolean
+    name: 'snapshotDirectory'
+    description: |
+      If enabled, a NFS volume will contain a read-only .snapshot directory which provides access to each of the volume's snapshots. Will enable "Previous Versions" support for SMB.
+  - !ruby/object:Api::Type::String
+    name: 'usedGib'
+    description: |
+      Used capacity of the volume (in GiB). This is computed periodically and it does not represent the realtime usage.
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'securityStyle'
+    description: |
+      Security Style of the Volume. Use UNIX to use UNIX or NFSV4 ACLs for file permissions.
+      Use NTFS to use NTFS ACLs for file permissions. Can only be set for volumes which use SMB together with NFS as protocol.
+    values:
+      - :NTFS
+      - :UNIX
+    immutable: true
+    default_from_api: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'kerberosEnabled'
+    description: |
+      Flag indicating if the volume is a kerberos volume or not, export policy rules control kerberos security modes (krb5, krb5i, krb5p).
+    immutable: true
+    default_value: false
+  - !ruby/object:Api::Type::Boolean
+    name: ldapEnabled
+    description: |
+      Flag indicating if the volume is NFS LDAP enabled or not. Inherited from storage pool.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'activeDirectory'
+    description: |
+      Reports the resource name of the Active Directory policy being used. Inherited from storage pool.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'kmsConfig'
+    description: |
+      Reports the CMEK policy resurce name being used for volume encryption. Inherited from storage pool.
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'encryptionType'
+    description: |
+      Reports the data-at-rest encryption type of the volume. Inherited from storage pool.
+    values:
+      - :ENCRYPTION_TYPE_UNSPECIFIED
```

Comment:
_UNSPECIFIED value probably should be removed

Reply:
This is an output:true field. The user cannot send the value.